clients/thea/src/lib.rs

```diff
@@ -171,10 +176,28 @@ where
 		protocol_name,
 		metrics,
 		_marker: Default::default(),
-		foreign_chain: Arc::new(foreign_connector),
+		foreign_chain: foreign_connector,
 	};
 
 	let worker = ObWorker::<_, _, _, _, _, _, _>::new(worker_params).await;
 
 	worker.run().await
 }
+
+pub struct Connector {
+	connector: Arc<dyn ForeignConnector>,
+}
+
+pub async fn get_connector(chain_type: ChainType) -> Connector {
+	log::info!(target:"thea","Assigning connector based on chain type: {:?}",chain_type);
+	match chain_type {
+		ChainType::Development => Connector { connector: Arc::new(NoOpConnector) },
+		_ => Connector {
+			connector: Arc::new(
+				ParachainClient::connect("ws://127.0.0.1:9945".to_string())
```

Comment:
As mentioned [here](https://github.com/Polkadex-Substrate/Polkadex/pull/709#discussion_r1173354847) - I think such values should not be hardcoded.